bluesky/run_engine.py

```diff
@@ -215,6 +215,12 @@ def block(ev: Threading.Event) -> None:
            - Matplotlib is imported and using a nbagg or ipympl backend (
              wait on the event and poll to push updates to the browser)
 
+    call_returns_uid: bool, optional, default True
```

Comment:
Maybe do this as 

```
call_return_type : str, optional {'uids', 'plan_return'}
```

which is a bit more "self documenting" and does not paint us into a corner if in the future we decide we want a third different return type!

Reply:
What would the default behavior be if the user passed something different from `'uids'` or `'plan_return'`? I can think of two options -- either we check `call_return_type in ['uids', 'plan_return']` at the top of the initialization and throw an error if something different is passed in, or we do
```
if call_return_type == 'plan_return':
    return plan_result
else:
    return tuple(self._run_start_uids)
```
and just make returning a uid list the default for any string value besides `'plan_return'`